examples/native-hybrid-app/ios/iOS Flutter Hybrid Example/FirstViewController.swift

```diff
@@ -14,9 +15,29 @@ class FirstViewController: UIViewController {
     }
     
     @IBAction func openFlutterView(_ sender: Any) {
-        let flutterViewController =
-            FlutterViewController(project: nil, nibName: nil, bundle: nil)
-        GeneratedPluginRegistrant.register(with: flutterViewController.engine!);
+        let flutterEngine = (UIApplication.shared.delegate as! AppDelegate).flutterEngine
+        let flutterViewController = FlutterViewController(engine: flutterEngine, nibName: nil, bundle: nil)
+        flutterViewController.modalPresentationStyle = .overFullScreen
+        flutterViewController.pushRoute("/")
+        
+        DismissMethodCallHandler.shared.setDismissListener {
+            self.dismiss(animated: true) {
+                Global.rum.startView(viewController: self)
+            }
```

Comment:
Note that it leaks `self` as we capture a strong reference in dismiss listener. Using `[weak self]` could solve the problem if it's not intended.

Reply:
Probably not an issue as the block gets discarded once it's called, but definitely better practice to hold onto it weak.